masthead.html

```
@@ -0,0 +1,360 @@
+
+
+<!-- Admin -->
+<div class="section">
+
+<h2 class="section-title">👨‍💻 Project Admin</h2>
+
+<div class="grid">
+
+  <div class="card admin-card">
+
+    <img src="https://avatars.githubusercontent.com/u/111553589?v=4">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ img ] must be self closed.

(tag-self-close)